tests/conftest.py

```diff
@@ -1,10 +1,15 @@
-"""
-    Dummy conftest.py for template_agent.
+"""Pytest fixture for the whois domain agent."""
+import pytest
 
-    If you don't know what this is for, just leave it empty.
-    Read more about conftest.py under:
-    - https://docs.pytest.org/en/stable/fixture.html
-    - https://docs.pytest.org/en/stable/writing_plugins.html
-"""
+from ostorlab.agent import definitions as agent_definitions
+from ostorlab.runtimes import definitions as runtime_definitions
+from agent import whois_domain_agent
 
-# import pytest
+
+@pytest.fixture
+def whois_test_agent():
+    """Creates a dummy agent for the Whois Domain Agent.
+    """
+    agent_definition = agent_definitions.AgentDefinition(name='whois')
```

Comment:
Create agent definition from the yaml file, you can see example in the zap agent.